### PR TITLE
Feature#11 create covid tests

### LINKED ISSUE #11: Create covid tests
# REQUIREMENTS
Create the covid test table and add values.
These tests can be sent through the API

# DESIGN
Provide references to images of the design, how it looks like now, and how it should look like.

# Acceptance Criteria
List all required acceptance criteria
- [x] Create all types of covid tests
- [x] Create endpoints for covid test
- [x] Unit test controller and request

### PR DESCRIPTION
**Pull request by mariormt17**
# Description
What did you implemented/Why did you implemented this:
 - In this PR I added the list of types of covid tests in the web project and API, these were added in the seeds folder.
 - The endpoints was created on `api/v1/covid_tests`
 - These changes are necessary to share the data with another team.
 Examples:
 - With these changes the covid test types can be consulted by the RN team through the API, as well as being used on the website.
 
 # Testing
I create unit test for request and routing.
you can test run `bundle exec rspec`
  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes
Captura de Pantalla 2021-03-30 a la(s) 12 52 37 p m

### CODE REVIEW

File: alerta-covid/app/assets/stylesheets/covid_tests.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the CovidTests controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (ArielMD) [inline alerta-covid/app/assets/stylesheets/covid_tests.scss]: you can delete this file

File: alerta-covid/app/assets/stylesheets/scaffolds.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  margin: -7px -7px 0;
+  background-color: #c00;
+  color: #fff; }
+
+#error_explanation ul li {
+  font-size: 12px;
+  list-style: square; }
+
+label {
+  display: block; }
```
Comment (ArielMD) [inline alerta-covid/app/assets/stylesheets/scaffolds.scss]: you can delete this file

**Review by ArielMD — DISMISSED** [review]
Nice job, are some files that you can delete, but looks good

**Review by ArielMD — DISMISSED** [review]
Nice

File: alerta-covid/spec/helpers/covid_tests_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Specs in this file have access to a helper object that includes
+# the CovidTestsHelper. For example:
+#
+# describe CovidTestsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
Comment (MiguelOcegueraM) [inline alerta-covid/spec/helpers/covid_tests_helper_spec.rb:12]: remember delete commented code if you don't need it

**Review by MiguelOcegueraM — DISMISSED** [review]
Nice code dudes! very clean

**Review by ramasoftdev — DISMISSED** [review]
All OK

**Review by ArielMD — APPROVED** [review]

**Pull request merged**